config/routes.rb

```diff
@@ -0,0 +1,8 @@
+Rails.application.routes.draw do
+  root to: 'application#root'
+  get 'restaurants/options', to: 'restaurants#options', defaults: { format: 'json' }
+  resources :restaurants, defaults: { format: 'json' } do
+    get :index
+    post :create
```

Comment:
did you try adding here
```
resources :restaurants, defaults: { format: 'json' } do
    get :index
    post :create
    member do
       get :options
    end
end
```
instead of `get 'restaurants/options', to: 'restaurants#options', defaults: { format: 'json' }` ? That should work and be cleaner